src/screens/TopicPageScreen/index.js

```diff
@@ -30,6 +29,7 @@ import {getUserId} from '../../utils/users';
 import {linkContextScreenParamBuilder} from '../../utils/navigation/paramBuilder';
 import {normalize, normalizeFontSizeByWidth} from '../../utils/fonts';
 import {setFeedByIndex, setTopicFeedByIndex, setTopicFeeds} from '../../context/actions/feeds';
+import BottomSheetFollow from './elements/BottomSheetFollow';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new component `BottomSheetFollow` has been imported. Ensure that this component is properly tested and follows the project's coding standards.
```diff
+ import BottomSheetFollow from './elements/BottomSheetFollow';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->